tests/slow/src/test/scala/tests/feature/Scala3CodeActionLspSuite.scala

```diff
@@ -495,6 +495,45 @@ class Scala3CodeActionLspSuite
     selectedActionIndex = 1,
   )
 
+  checkExtractedMember(
+    "extract-object-main-annot",
+    """|package a
+       |@main def buildSite() =  ???
```

Comment:
Does it actually matter if main annotation is used? If we have any def we should treat the file as one that might contain some toplevel methods. What do you think @ckipp01 ?

Reply:
Yea exactly, even if you have something incredibly simple with no main method like:

```
package example

given String = "hello"

object Foo
```

You should be able tot move `Foo` out into a `Foo.scala`.